fastly/block_fastly_service_backend.go

```diff
@@ -276,6 +282,7 @@ func (h *BackendServiceAttributeHandler) buildCreateBackendInput(service string,
 		ErrorThreshold:      gofastly.Int(resource["error_threshold"].(int)),
 		FirstByteTimeout:    gofastly.Int(resource["first_byte_timeout"].(int)),
 		HealthCheck:         gofastly.String(resource["healthcheck"].(string)),
+		KeepAliveTime:       gofastly.Int(resource["keepalive_time"].(int)),
```

Comment:
We should probably only set this field on `opts` if the value provided is greater than zero. I don't think it makes sense to send zero (which is what it will be if the user doesn't define it in their config).

Reply:
So remove the field from here and underneath the `opts` declaration/assignment have something like...

```go
if resource["keepalive_time"].(int) > 0 {
    opts. KeepAliveTime = gofastly.Int(resource["keepalive_time"].(int))
}
```